.gitignore

```diff
@@ -7,9 +7,19 @@
 *.png
 *.log
 *.pdf
+*.txt
+
+*.fits
+
+notes.txt
+
+mynotebooks/
```

Comment:
What I did was to run: 
```
chmod -R -x+X * 
```
In the `notebooks/` directory, I hope this solves the issue.